calebwagner/python-server
# Cjw ch11

#### Changes Made
1. Modified file `animals/request.py` to include `delete_animal` function.
1. Modified file `request_handler.py` to include `do_DELETE` function.
   ​
#### Steps to Review
1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout cjw-ch11
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality:
- run debugger server
- from the Postman client, perform a DELETE request to http://localhost:8088/animals/5
- then request all animals again with a GET request to http://localhost:8088/animals
- verify that the animal with an id of 5 is not in the response and got deleted from the database.

4. View code file.
   > Confirm file modifications are present as indicated above.
